src/chainstate/coordinator/mod.rs

```diff
@@ -407,7 +409,7 @@ pub fn get_reward_cycle_info<U: RewardSetProvider>(
             }));
         }
 
-        info!("Beginning reward cycle. block_height={}", burn_height);
+        info!("Beginning reward cycle. block_height={}, reward_cycle_length={}, prepare_phase_length={}", burn_height, burnchain.pox_constants.reward_cycle_length, burnchain.pox_constants.prepare_length);
```

Comment:
Can you update this `info!` log to use keys/values format:

```suggestion
        info!("Beginning reward cycle";
              "burn_height" => burn_height,
              "reward_cycle_length" => burnchain.pox_constants.reward_cycle_length,
              "prepare_phase_length" => burnchain.pox_constants.prepare_length);
```

Reply:
Should be fixed now.